kratos/sources/model_part.cpp

```diff
@@ -1786,6 +1786,9 @@ void ModelPart::PrintData(std::ostream& rOStream) const
     rOStream << "    Number of tables : " << NumberOfTables() << std::endl;
     rOStream << "    Number of sub model parts : " << NumberOfSubModelParts() << std::endl;
     if (!IsSubModelPart()) {
+        if (IsDistributed()) {
+        rOStream << "    Distributed over " << mpCommunicator->TotalProcesses() << " processes" << std::endl;
```

Comment:
Well this prints the total number of ranks. But all the ranks may not have 'this' modelpart. I find printing this is a bit misleading !